x-pack/plugins/cases/server/telemetry/constants.ts

```diff
@@ -0,0 +1,11 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+/**
+ * This should only be used within telemetry
+ */
+export const OWNERS = ['observability', 'securitySolution', 'cases'] as const;
```

Comment:
If I remember correctly we have these constants defined in the `common/constant.ts` file, correct?